.github/workflows/check-and-register.yml

```
@@ -8,8 +8,8 @@ on:
 
 jobs:
   build:
-    uses: octopusden/octopus-base/.github/workflows/common-check-and-register-release.yml@v2.1.8
+    uses: octopusden/octopus-base/.github/workflows/common-check-and-register-release.yml@v2.1.12
     if:  "${{ github.event.workflow_run.conclusion == 'success' }}"
```

YAMLlint (1.38.0)

[error] too many spaces after colon

(colons)